source/text/Format.cpp

```diff
@@ -298,7 +298,7 @@ string Format::Capitalize(const string &str)
 	bool first = true;
 	for(char &c : result)
 	{
-		if(!isalpha(c))
+		if(c == ' ')
```

Comment:
Rather than just _space_, I would argue for things like tab or newline, too.
```suggestion
		if(isspace(c))
```
(We should really also cast `c` to `unsigned char` before using these functions, to avoid undefined behavior, but that's something to be addressed more fully by @oo13 in #4123.)

Reply:
FYI
I pushed the fix (6fbd9743a292c50668401bb9028b9f41137246b2) in #4123.
It's picked from my translatable branch.